Handler/EmailHandler.php

```diff
@@ -35,6 +36,8 @@ public function handle(SubmissionConfig $submission, FormInterface $form, FormCo
                 ->setFrom($email->getFrom())
                 ->setTo($email->getEndpoint())
                 ->setBody($email->getBody());
+
+            $message = $this->addAttachments($message, $email->getAttachments());
```

Comment:
$message is an object passed by reference. You should not need to return the $message object in the addAttachments function ?